seacatauth/apikey/handler.py

```
@@ -0,0 +1,142 @@
+import logging
+import asab
+import asab.web.rest
+import asab.web.auth
+import asab.web.tenant
+import asab.exceptions
+
+from .. import generic, exceptions
```

Ruff (0.12.2)

`..exceptions` imported but unused

Remove unused import: `..exceptions`

(F401)